lib/core/Head.js

```diff
@@ -29,6 +29,11 @@ class Head extends React.Component {
           highlight.version
         }/styles/${highlight.theme}.min.css`;
 
+    // ensure the siteUrl variable ends with a single slash
+    const siteUrl =
+      (this.props.config.url + this.props.config.baseUrl).replace(/\/+$/, '') +
+      '/';
```

Comment:
So this won't cause a possible double ending `//` when we use it below?

For example, `siteUrl` always ends with a single slash. Could `this.props.config.ogImage` start with a single slash, causing a `//` in line 51?

Just wanting to make sure we are still good there.